python/py-spaCy/Portfile

```diff
@@ -0,0 +1,72 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem                                      1.0
+PortGroup           python                      1.0
+PortGroup           cxx11                       1.1
+PortGroup           github                      1.0
+PortGroup           mpi                         1.0
+PortGroup           compiler_blacklist_versions 1.0
+
+name                py-spaCy
+version             2.1.0a13
+revision            0
+github.setup        explosion spaCy ${version} v
+
+checksums           rmd160  0c59088115a90f5590eee9cccc6b1bed3805f51d \
+                    sha256  74aef9a26c7e22988a1590a9f6fe14f66504742c0e096aa9873370f31dcb17e9 \
+                    size    27225747
+
+platforms           darwin
+supported_archs     x86_64
+
+license             MIT
+
+maintainers         {jonesc @cjones051073} openmaintainer
+
+description         Industrial-strength Natural Language Processing (NLP) with Python and Cython
+long_description    ${description}
+
+homepage            https://spacy.io
+
+# Support python versions
+python.versions     35 36 37
+
+mpi.setup           -gcc44 -gcc45 -clang33 -clang34 -clang37 -clang38 -clang39 -clang40 -gfortran -g95
+
+# Compiler selection
+compiler.blacklist-append *gcc* {clang < 800} macports-clang-3.3 macports-clang-3.4 \
+                          macports-clang-3.7 macports-clang-4.0 macports-clang-3.9
+compiler.whitelist clang macports-clang-7.0 macports-clang-6.0 macports-clang-5.0
+
+if {${name} ne ${subport}} {
+
+    set PythonVersionWithDot [join [split ${python.version} ""] "."]
+
+    depends_build-append \
+        port:cctools \
+        port:py${python.version}-pytest \
+        port:py${python.version}-setuptools \
+        port:py${python.version}-wheel
+
+    depends_lib-append \
```

Comment:
done

Reply:
almost... pahtlib2 is not needed (since you don't have Python versions below 3.4) and should be removed. In addition, regex/mock/flake8 (now present twice) are not listed as dependencies, but are only needed for tests and should be removed from depends_lib. I am not sure why regex/pip are listed here, but they're not in setup.py either. Ideally you would test whether these are actually needed instead of just adding them here..